docs/sections/user_guide/cli/drivers/orog_gsl/help.out

```diff
@@ -12,6 +12,8 @@ Optional arguments:
 
 Positional arguments:
   TASK
+    input_config_file
+      The input config file
```

Comment:
Here and in the `shave` driver, the run command was causing problems bc the program said it could read Unit 5 from the file in `/tmp`. I went with the `exec < config_file.in` type approach to the run command, so added the creation of the config file for each of those drivers.